deno_webgpu/device.rs

```diff
@@ -134,15 +135,12 @@ impl GPUDevice {
 
     #[required(1)]
     #[cppgc]
-    fn create_buffer(
-        &self,
-        #[webidl] descriptor: super::buffer::GPUBufferDescriptor,
-    ) -> Result<GPUBuffer, JsErrorBox> {
+    fn create_buffer(&self, #[webidl] descriptor: super::buffer::GPUBufferDescriptor) -> GPUBuffer {
         let wgpu_descriptor = wgpu_core::resource::BufferDescriptor {
             label: crate::transform_label(descriptor.label.clone()),
             size: descriptor.size,
             usage: wgpu_types::BufferUsages::from_bits(descriptor.usage)
-                .ok_or_else(|| JsErrorBox::type_error("usage is not valid"))?,
+                .unwrap_or(wgpu_types::BufferUsages::empty()),
```

Comment:
is this correct?
the spec states:
```
descriptor.usage must be a subset of the allowed buffer usages for this.
```

Reply:
The issue is that the validation needs to happen on the device timeline, i.e., the error should be raised through `popErrorScope()` or `onuncapturederror`, rather than being thrown immediately. wgpu will still reject `BufferUsages::empty()`, this just changes when/how the error is raised, which the CTS tests were sensitive to.

I added a comment explaining this.